solr/test-framework/src/java/org/apache/solr/cloud/MiniSolrCloudCluster.java

```diff
@@ -1341,5 +1349,10 @@ public Builder withDefaultClusterProperty(String key, String value) {
       cluster.put(key, value);
       return this;
     }
+
```

Comment:
Can you please add javadoc here; it may not be obvious what it does and why a test might want to set it.
I'm thinking maybe:
> Disables the default/built-in simple trace ID generation/propagation.  For example if you want to use a  MockTracer or a "real" Tracer, you'll need to call this so that it doesn't displace the Tracer you want to use.